### ISSUE TITLE
Improve accessibility for adding training modules on a block

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#2505]
Currently, when a user adds training modules to a block, they show up in a multi-select.
![image](https://user-images.githubusercontent.com/1316902/53052584-f347a700-3453-11e9-9cd8-c6ad39a10640.png)

However, the close buttons for each individual item are not accessible. We need to label these so a screen reader can read off individual items for removal.

**Comment by ragesoss** [comment]
I note that the new react-select release that I just merged has several a11y fixes. Once it's deployed, we should see if this is already fixed for Helaine.

https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/2528

**Comment by ragesoss** [comment]
Nope, the latest release didn't change it for Helaine.
